src/runtime-rs/crates/runtimes/src/direct_volume/mod.rs

```diff
@@ -0,0 +1,156 @@
+use anyhow::{anyhow, Ok, Result};
+use base64;
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+use std::{collections::HashMap, path::PathBuf};
+
+const KATA_DIRECT_VOLUME_ROOT_PATH: &str = "/run/kata-containers/shared/direct-volumes";
+const MOUNT_INFO_FILE_NAME: &str = "mountInfo.json";
+
+pub const DIRECT_VOLUME_STAT_URL: &str = "/direct-volume/stats";
+pub const DIRECT_VOLUME_RESIZE_URL: &str = "/direct-volume/resize";
+pub const DIRECT_VOLUME_PATH_KEY: &str = "path";
+
+#[derive(Serialize, Deserialize)]
+pub struct VolumeMountInfo {
```

Comment:
As this struct needs to be used by both `runtime-rs` and `kata-ctl`, I think it's better to put it under `libs/kata-types/src/mount.rs`, and rename it to `DirectVolumeMountInfo`.